.travis.yml

```diff
@@ -51,6 +51,12 @@ jobs:
   - env: TEST_TYPE=upgrade CASSANDRA_VERSION=git:trunk CO="-t @all_nodes_reachable"
   - env: TEST_TYPE=upgrade CASSANDRA_VERSION=git:trunk CO="-t ~@all_nodes_reachable"
   include:
+    - stage: Integration Tests
+      name: Memory, H2 and Postgresql
```

Comment:
Each name also should be updated, to include Cassandra version, so it's obvious which build is succeeding/failing. (my bad for not seeing this before.)

Reply:
Definitely make sense :+1: